### PR TITLE
Italian translation

### PR DESCRIPTION
**Pull request by ErikPelli**
I translated the strings.xml file in italian (in some parts there might be some little error but it's ok).

### CODE REVIEW

File: Mifare Classic Tool/app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+    <string name="text_strange_error">È avvenuto un errore sconosciuto :(</string>
+    <string name="text_invalid_ac_or_sector_dead">Condizioni di accesso non valide o settore morto :(</string>
+    <string name="text_keys_read_only">Le chiavi (A e B) sono di sola lettura</string>
+    <string name="text_ac_read_only">Le condizioni di accesso sono di sola lettura</string>
+    <string name="text_write_key_b_not_known">Chiave col permesso di scrivere (B)         sconosciuta</string>
+    <string name="text_write_key_a_not_known">Chiave col permesso di scrivere (A)         sconosciuta</string>
+    <string name="text_key_map_progress">Progresso nella mappatura delle chiavi:</string>
+    <string name="title_activity_data_conversion_tool">Strumento per la Conversione dei Dati</string>
+    <string name="text_ac_incr">Incr.:</string>
+    <string name="text_identical_data">-----------Identico------------</string>
```
Comment (ikarus23) [inline Mifare Classic Tool/app/src/main/res/values-it/strings.xml:82]: Is not 32 chars long

File: Mifare Classic Tool/app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+    <string name="text_chooser_info_text">Scegli un file:</string>
+    <string name="text_no_files_in_chooser">Nessun file trovato</string>
+    <string name="text_open_dump_title">Apri dump di un tag</string>
+    <string name="text_ac_pos">Posizione:</string>
+    <string name="text_vb_location">Posizione:</string>
+    <string name="text_strange_error">È avvenuto un errore sconosciuto :(</string>
+    <string name="text_invalid_ac_or_sector_dead">Condizioni di accesso non valide o settore morto :(</string>
+    <string name="text_keys_read_only">Le chiavi (A e B) sono di sola lettura</string>
+    <string name="text_ac_read_only">Le condizioni di accesso sono di sola lettura</string>
+    <string name="text_write_key_b_not_known">Chiave col permesso di scrivere (B)         sconosciuta</string>
```
Comment (ikarus23) [inline Mifare Classic Tool/app/src/main/res/values-it/strings.xml:77]: Why are there so much spaces?

**Comment by ikarus23** [comment]
Hi! Thank you for this contribution! I found some strange lines and comment on them. Could you check them out?

File: Mifare Classic Tool/app/src/main/res/values-it/strings.xml (hunk 1 above)
Comment (ErikPelli) [reply Mifare Classic Tool/app/src/main/res/values-it/strings.xml]: fixed

File: Mifare Classic Tool/app/src/main/res/values-it/strings.xml (hunk 2 above)
Comment (ErikPelli) [reply Mifare Classic Tool/app/src/main/res/values-it/strings.xml]: Sorry, android studio view \n as spaces, I missed this line in the review.

**Comment by ErikPelli** [comment]
> Hi! Thank you for this contribution! I found some strange lines and comment on them. Could you check them out?

I think I've fixed them

**Comment by ikarus23** [comment]
I can also see stuff like
`<string name="text_tag_type_guess">\Il campo "Produttore e tipo di tag\" potrebbe`. What is with `\Il` at the beginning?

**Comment by ErikPelli** [comment]
I'm sorry I wasted your time.
I checked the entire file and I hope I have fixed all the problems you reported (and any others that were there).

**Comment by ikarus23** [comment]
Don't worry! I'm grateful for the contribution and your patience to bring everything i a format were I'm happy to merge it. :+1: 
I review it tomorrow. But I think it looks good now.

**Comment by ErikPelli** [comment]
> Don't worry! I'm grateful for the contribution and your patience to bring everything i a format were I'm happy to merge it. +1
> I review it tomorrow. But I think it looks good now.

👍🏻

**Comment by ErikPelli** [comment]
Can you wait the next commit for the review? I realized I forgot to fix something important.

**Comment by ikarus23** [comment]
Sure!

**Comment by ErikPelli** [comment]
Now everything should be ok

**Comment by ikarus23** [comment]
Looks great! Thanks again!

**Pull request merged**